.travis.yml

```diff
@@ -1,24 +1,18 @@
 language: cpp
 
 before_install:
-  - if [ "$INSTALL" = "true" ]; then
-      sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test;
-      sudo apt-get update -qq;
-      sudo apt-get install -qq -y --force-yes gcc-4.8 g++-4.8;
-    fi
   - if [ "$OS" = "OSX" ]; then
       brew update;
       brew upgrade;
     fi
 
 install:
-    - if [ "$EXPORT" = "true" ]; then
-        if [ "$CXX" = "g++" ]; then export CXX="g++-$CXX_V" CC="gcc-$CC_V"; fi;
-        if [ "$CXX" = "clang++" ]; then
-          export CXX="clang++-$CXX_V" CC="clang-$CC_V";
-        fi;
-      fi
-    - ./autogen.sh
+    - if [ "$CXX" = "g++" -a -n "${VER+1}" ]; then 
+          export CXX="g++-$VER" CC="gcc-$VER"; 
+      fi;
+    - if [ "$CXX" = "clang++" -a -n "${VER+1}" ]; then
+          export CXX="clang++-$VER" CC="clang-$VER";
+      fi;
```

Comment:
I really don't like the way in which this is implemented! Why don't we directly export `CXX` and `CC`? If it's not possible to specify that variables directly, why don't use `MK_CXX` and `MK_CC` to override `CXX` and `CC` in the proper place of the Valgrind file?